django/text_extractor/utils.py

```diff
@@ -285,3 +285,8 @@ def create_searchable_pdf(input_file, add_header):
         output.add_page(new_pdf_page.pages[0])
 
     return output, all_text
+
+
+def shorten_input_name(input_name):
+    base_name, file_extension = os.path.splitext(input_name)
+    return str(uuid.uuid4()) + file_extension
```

Comment:
Use a UUID for file to ensure uniqueness and length (no need to prepend username and date etc.)